x/ibc/07-tendermint/client/cli/tx.go

```diff
@@ -170,3 +184,25 @@ $ %s tx ibc client misbehaviour [path/to/evidence.json] --from node0 --home ../n
 
 	return cmd
 }
+
+func parseFraction(fraction string) (tmmath.Fraction, error) {
+	fr := strings.Split(fraction, "/")
+	if len(fr) == 0 || len(fr) > 2 || fr[0] == fraction {
```

Comment:
```suggestion
	if len(fr) != 2 || fr[0] == fraction {
```

Reply:
tbh, I don't quite get the `fr[0] == fraction` check.

Reply:
> If s does not contain sep and sep is not empty, Split returns a
slice of length 1 whose only element is s.